base/resources/scripts/backup_restore_config.py

```diff
@@ -56,6 +56,7 @@
     backup_selection = "--include='/.config' \
                         --include='/.config/xfce4/' --include='/.config/xfce4/xfconf/***' \
                         --include='/.config/Code/' --include='/.config/Code/User/' --include='/.config/Code/User/settings.json' \
+                        --include='/.config/Code/User/argv.json' --include='/.config/Code/languagepacks.json'\
```

Comment:
I don't know what this backup stuff is, but I'm wondering: should everything under `Code/` be included via wildcard pattern?

Reply:
I'm also not 100% sure on the backup stuff but I can say that we can include everything under `Code/` without breaking anything. The files that are included are all JSON files that just configure vscode. 
The change will be replacing those two lines with
`--include='/.config/Code/***' \` as rsync's include pattern requires triple asterisk for said directory and everything under it